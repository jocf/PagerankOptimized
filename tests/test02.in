1
0.85
1
A
0
