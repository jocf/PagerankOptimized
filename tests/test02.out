A 0.1500
